Flepper.Tests.Unit/QueryBuilder/Commands/WhereFilterTests.cs

```diff
@@ -242,60 +242,87 @@ public void ShoulContainWhereWithNotEqualToNull()
         [Fact]
         public void ShoulContainLikeContains()
         {
-            FlepperQueryBuilder.Select()
+            QueryResult result = FlepperQueryBuilder.Select()
                 .From("table")
                 .Where("field").Contains("abc")
-                .Build()
+                .BuildWithParameters();
+
+            result.Query
                 .Trim()
                 .Should()
                 .Contain("LIKE");
+
+            dynamic parameters = result.Parameters;
+            Assert.Equal("%abc%", parameters.@p0);
         }
 
         [Fact]
         public void ShoulContainLikeStartsWith()
         {
-            FlepperQueryBuilder.Select()
+            QueryResult result = FlepperQueryBuilder.Select()
                 .From("table")
                 .Where("field").StartsWith("abc")
-                .Build()
+                .BuildWithParameters();
+
+            result.Query
                 .Trim()
                 .Should()
                 .Contain("LIKE");
+
+            dynamic parameters = result.Parameters;
+            Assert.Equal("%abc", parameters.@p0);
         }
 
         [Fact]
         public void ShoulContainLikeEndsWith()
         {
-            FlepperQueryBuilder.Select()
+            QueryResult result = FlepperQueryBuilder.Select()
                 .From("table")
                 .Where("field").EndsWith("abc")
-                .Build()
+                .BuildWithParameters();
+
+            result.Query
                 .Trim()
                 .Should()
                 .Contain("LIKE");
+
+            dynamic parameters = result.Parameters;
+            Assert.Equal("abc%", parameters.@p0);
         }
 
         [Fact]
         public void ShoulContainLikeContainsAndStartsWithAndEndsWith()
         {
-            var result = FlepperQueryBuilder.Select()
+            QueryResult result = FlepperQueryBuilder.Select()
                 .From("table")
-                .Where("field1").Contains("abc")
-                .And("field2").StartsWith("abc")
-                .And("field3").EndsWith("abc")
-                .Build()
+                .Where("field1").Contains("abc1")
+                .And("field2").StartsWith("abc2")
+                .And("field3").EndsWith("abc3")
+                .BuildWithParameters();               
+
+            result.Query.Should()
+                .Contain("@p0");
+            result.Query.Should()
+                .Contain("@p1");
+            result.Query.Should()
+                .Contain("@p2");
+
+            result.Query.Should()
+                .Contain("[field1]");
+            result.Query.Should()
+                .Contain("[field2]");
```

Comment:
You still must test the FULL SQL Query, do not use Contains

Reply:
I did not understand

Reply:
Ok, Test done